### PR TITLE
Show contributers avatars in contributers tableView

### PR DESCRIPTION
**Pull request by mahmoudfotta**
*add avatarUrl to contributer struct
*decode contributer struct using codingkeys
*add avatarImageView to ContributerCell
*send avatarURL to avatarImageView to load image

### CODE REVIEW

**Comment by Yonodactyl** [comment]
@kgellci Any chance you can check this?

**Comment by kgellci** [comment]
Sorry, missed this, looking now.

File: Area51/Sources/ContributionsController.swift [hunk 1]
```diff
@@ -32,17 +32,19 @@ class ContributionController: UITableViewController {
...
-
     override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         let cell: ContributorCell = tableView.reusableCell(forIdentifier: ContributorCell.reuseIdentifier)
         let contributors = listOfContributors[indexPath.item]
 
         cell.contributorName.text = contributors.login ?? contributors.name
         cell.contributionsLabel.text = "\(contributors.contributions) Contributions"
 
+        cell.avatarImageView.isHidden = contributors.avatarURL == nil
+        if let avatarURL = URL(string: contributors.avatarURL ?? "") {
```
Comment (kgellci) [inline Area51/Sources/ContributionsController.swift]: You can replace this if else statement with
`cell.avatarImageView.url = contributors.avatarURL.flatMap(URL.init)`

File: Area51/Sources/ViewModels/ContributorModel.swift [hunk 2]
```diff
@@ -5,6 +5,20 @@ struct Contributor: Decodable {
...
     var name: String?
     var contributions: Int
+    var avatarURL: String?
+
+    enum CodingKeys: String, CodingKey {
+        case login, name, contributions
+        case avatarURL = "avatar_url"
+    }
+
+    init(from decoder: Decoder) throws {
```
Comment (kgellci) [inline Area51/Sources/ViewModels/ContributorModel.swift]: can delete this entire init block, the coding should just work with the CodingKeys specified above.

**Comment by kgellci** [comment]
@mahmoudfotta Thanks for adding, I left a couple comments, should be good to go once those are resolved.

**Comment by mahmoudfotta** [comment]
fixed comments and pushed @kgellci

**Review by kgellci — APPROVED** [review]
Awesome! Feel free to update the changelog with this change.

**Pull request merged**